plugins/dev-scripts/config/webpack.config.js

```diff
@@ -120,7 +118,7 @@ module.exports = (env) => {
           enforce: 'pre',
         },
         // Run babel to compile both JS and TS.
```

Comment:
Please remove this comment (since it should be clear from the actual code). Otherwise update it to be correct.

I would also be in favor of moving the typescript case above the JS case.

Reply:
Done!

Reply:
Moved the TS case above the JS case!